From a0ba8715ada99df6aa9076bb7b189af8a8ffc6b7 Mon Sep 17 00:00:00 2001
From: Ranjani Sridharan <ranjani.sridharan@linux.intel.com>
Date: Wed, 20 Jun 2018 15:22:43 -0700
Subject: [PATCH 1223/1586] ASoC: uapi: sof: add token for SSP MCLK ID

Add a token for SSP_MCLK_ID which will be used to pass the MCLK ID
for configuring the SSP.

Signed-off-by: Ranjani Sridharan <ranjani.sridharan@linux.intel.com>
---
 include/uapi/sound/sof-topology.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/uapi/sound/sof-topology.h b/include/uapi/sound/sof-topology.h
index 1a4957b..721a2b5 100644
--- a/include/uapi/sound/sof-topology.h
+++ b/include/uapi/sound/sof-topology.h
@@ -68,6 +68,7 @@
 #define SOF_TKN_INTEL_SSP_MCLK_KEEP_ACTIVE	500
 #define SOF_TKN_INTEL_SSP_BCLK_KEEP_ACTIVE	501
 #define SOF_TKN_INTEL_SSP_FS_KEEP_ACTIVE	502
+#define SOF_TKN_INTEL_SSP_MCLK_ID		503
 
 /* DMIC */
 #define SOF_TKN_INTEL_DMIC_DRIVER_VERSION	600
-- 
2.7.4

